Shared/Samples/Style point with picture marker symbols/StylePointWithPictureMarkerSymbolsView.swift

```diff
@@ -0,0 +1,103 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct StylePointWithPictureMarkerSymbolsView: View {
+    /// A graphics overlay to hold the picture marker symbol graphics.
+    @State private var graphicsOverlay: GraphicsOverlay = {
+        let graphics = [
+            makePictureMarkerSymbolFromImage(),
+            makePictureMarkerSymbolFromURL()
+        ]
+        return GraphicsOverlay(graphics: graphics)
+    }()
+    
+    /// A map with topographic basemap and centered on Harman's Cross in England.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISTopographic)
+        map.initialViewpoint = Viewpoint(
+            center: Point(x: -225166.5, y: 6551249, spatialReference: .webMercator),
+            scale: 1e5
+        )
+        return map
+    }()
+    
+    /// Creates a picture marker symbol from an image in the project assets.
+    /// - Returns: A picture marker symbol
+    private static func makePictureMarkerSymbolFromImage() -> Graphic {
+        let imageName = "PinBlueStar"
+        
+        // Create pin symbol using the image.
+        let pinSymbol = PictureMarkerSymbol(image: UIImage(named: imageName)!)
+        
+        // Ensure the pin symbol loaded.
+        guard pinSymbol.loadStatus != .failed else {
+            return Graphic()
+        }
+        
+        // Change offsets, so the symbol aligns properly to the point.
+        pinSymbol.offsetY = pinSymbol.image!.size.height / 2
+        
+        // Create the location for pin.
+        let pinPoint = Point(x: -226773, y: 6550477, spatialReference: .webMercator)
+        
+        // Create the graphic for pin.
+        let pinGraphic = Graphic(geometry: pinPoint, symbol: pinSymbol)
+        
+        return pinGraphic
+    }
+    
+    /// Creates a picture marker symbol using a remote image.
+    /// - Returns: A picture marker symbol
+    private static func makePictureMarkerSymbolFromURL() -> Graphic {
+        let imageURL = URL(
+            string: "https://static.arcgis.com/images/Symbols/OutdoorRecreation/Camping.png"
+        )!
+        
+        // Create pin symbol using the URL.
+        let campsiteSymbol = PictureMarkerSymbol(url: imageURL)
+        
+        // FYI, for picture marker symbols created with the
+        // `PictureMarkerSymbol(url: URL)` initializer, the `image` property
+        // will be `nil` until the symbol is loaded using the
+        // `func load() async throws` method. We are not accessing that
+        // property here, so we don't need to explicitly load it. The symbol
+        // will be loaded automatically prior to drawing.
+        
+        // Ensure the campsite symbol loaded.
+        guard campsiteSymbol.loadStatus != .failed else {
+            return Graphic()
+        }
```

Comment:
Same here. You're not accessing `.image`, so no issues with the code without this guard.
```suggestion
```